### ISSUE TITLE
pcm_s8 codec not supported in WAVE format

### ISSUE THREAD

**Issue by thfalan** [jiaaro/pydub#417]
### Steps to reproduce
The module has been working perfectly for me until last week I encountered an error on a specific WAVE file, my code:

```
file = r"Dev.wav"
print(utils.mediainfo(file))
try:
    data = AudioSegment.from_file(file)

except Exception as e:
    print(e)
```

### Expected behavior
It should have read in the file

### Actual behavior
It threw an error:

> ffmpeg version 4.1.3 Copyright (c) 2000-2019 the FFmpeg developers
>   built with gcc 8.3.1 (GCC) 20190414
>   configuration: --disable-static --enable-shared --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --enable-libmfx --enable-amf --enable-ffnvcodec --enable-cuvid --enable-d3d11va --enable-nvenc --enable-nvdec --enable-dxva2 --enable-avisynth
>   libavutil      56. 22.100 / 56. 22.100
>   libavcodec     58. 35.100 / 58. 35.100
>   libavformat    58. 20.100 / 58. 20.100
>   libavdevice    58.  5.100 / 58.  5.100
>   libavfilter     7. 40.101 /  7. 40.101
>   libswscale      5.  3.100 /  5.  3.100
>   libswresample   3.  3.100 /  3.  3.100
>   libpostproc    55.  3.100 / 55.  3.100
> Guessed Channel Layout for Input Stream #0.0 : mono
> Input #0, wav, from 'NICE_Dev.wav':
>   Duration: 00:25:55.39, bitrate: 64 kb/s
>     Stream #0:0: Audio: pcm_mulaw ([7][0][0][0] / 0x0007), 8000 Hz, mono, s16, 64 kb/s
> Stream mapping:
>   Stream #0:0 -> #0:0 (pcm_mulaw (native) -> pcm_s8 (native))
> Press [q] to stop, [?] for help
> [wav @ 00000265db3c1bc0] pcm_s8 codec not supported in WAVE format
> Could not write header for output file #0 (incorrect codec parameters ?): Function not implemented
> Error initializing output stream 0:0 -- 
> Conversion failed!

### Your System configuration
- Python version: 3.7
- Pydub version: 0.23.1
- ffmpeg or avlib?: ffmpeg
- ffmpeg/avlib version: 1.4

### Is there an audio file you can include to help us reproduce?
You can include the audio file in this issue - just put it in a zip file and drag/drop the zip file into the github issue.

The audio file is confidential but the details of the file using utils.mediainfo is as follow:

> 
{'index': '0', 'codec_name': 'pcm_mulaw', 'codec_long_name': 'PCM mu-law / G.711 mu-law', 'profile': 'unknown', 'codec_type': 'audio', 'codec_time_base': '1/8000', 'codec_tag_string': '[7][0][0][0]', 'codec_tag': '0x0007', 'sample_fmt': 's16', 'sample_rate': '8000', 'channels': '1', 'channel_layout': 'unknown', 'bits_per_sample': '8', 'id': 'N/A', 'r_frame_rate': '0/0', 'avg_frame_rate': '0/0', 'time_base': '1/8000', 'start_pts': 'N/A', 'start_time': 'N/A', 'duration_ts': '12443128', 'duration': '1555.391000', 'bit_rate': '64000', 'max_bit_rate': 'N/A', 'bits_per_raw_sample': 'N/A', 'nb_frames': 'N/A', 'nb_read_frames': 'N/A', 'nb_read_packets': 'N/A', 'DISPOSITION': {'default': '0', 'dub': '0', 'original': '0', 'comment': '0', 'lyrics': '0', 'karaoke': '0', 'forced': '0', 'hearing_impaired': '0', 'visual_impaired': '0', 'clean_effects': '0', 'attached_pic': '0', 'timed_thumbnails': '0'}, 'filename': 'NICE_Dev.wav', 'nb_streams': '1', 'nb_programs': '0', 'format_name': 'wav', 'format_long_name': 'WAV / WAVE (Waveform Audio)', 'size': '12443174', 'probe_score': '99'}

Decoding failed. ffmpeg returned error code: 1

**Comment by c0bra** [comment]
@thfalan did you have any luck solving this?

**Comment by thfalan** [comment]
> @thfalan did you have any luck solving this?

Yes, it has been solved in #415 using an undocumented method - apparently it's something to do with stdin but my audio encoding knowledge is too limited to understand, appreciate if you could explain the problem.
